cpp/tests/datetime/datetime_ops_test.cpp

```diff
@@ -567,4 +567,39 @@ TEST_F(BasicDatetimeOpsTest, TestIsLeapYear)
       {true, false, true, true, true, true, true, true, false, true, true, false}});
 }
 
+TEST_F(BasicDatetimeOpsTest, TestQuarter)
+{
+  using namespace cudf::test;
+  using namespace cudf::datetime;
+  using namespace cuda::std::chrono;
+
+  // Time in seconds since epoch
+  // Dates converted using epochconverter.com
+  auto timestamps_s =
+    cudf::test::fixed_width_column_wrapper<cudf::timestamp_s, cudf::timestamp_s::rep>{
+      {
+        1594332839L,    // 2020-07-09 10:13:59 GMT
+        0L,             // null
+        915148800L,     // 1999-01-01 00:00:00 GMT
+        -11663029161L,  // 1600-5-31 05:40:39 GMT
+        707904541L,     // 1992-06-07 08:09:01 GMT
+        -2181005247L,   // 1900-11-20 09:12:33 GMT
+        0L,             // UNIX EPOCH 1970-01-01 00:00:00 GMT
+        -12212553600L,  // First full year of Gregorian Calandar 1583-01-01 00:00:00
+        0L,             // null
+        13591632822L,   // 2400-09-13 13:33:42 GMT
+        4539564243L,    // 2113-11-08 06:04:03 GMT
+        0L,             // null
+        1608581568L,    // 2020-12-21 08:12:48 GMT
+        1584821568L,    // 2020-03-21 08:12:48 GMT
+      },
+      {true, false, true, true, true, true, true, true, false, true, true, false, true, true}};
```

Comment:
Need to  add header `#include <cudf_test/iterator_utilities.hpp>` and `using namespace cudf::test::iterators;` before using this:
```suggestion
      nulls_at({1, 8, 11})};
```